sample.yaml

```diff
@@ -1,57 +0,0 @@
-apiVersion: k8s.wasmcloud.dev/v1alpha1
-kind: WasmCloudHostConfig
-metadata:
-  name: my-wasmcloud-cluster
-  namespace: default
-spec:
-  hostReplicas: 1
-  issuers:
-    - CDKF6OKPOBQKAX57UOXO7SCHURTOZWKWIVPC2HFJTGFXY5VJX44ECEHH
-  # The lattice to connect the hosts to
-  lattice: default
-  # Additional labels to apply to the host other than the defaults set in the controller
-  hostLabels:
-    test: value
-    cluster: kind
-  # Which wasmCloud version to use
-  version: "1.0.2"
-  # The name of a secret in the same namespace that provides the required secrets.
-  secretName: cluster-secrets
-  logLevel: INFO
-  natsAddress: nats://nats-cluster.default.svc.cluster.local
-  ################################################
-  # Additional options that can be set for hosts:
-  ################################################
-  # allowLatest: true
-  # allowedInsecure:
-  #   - "localhost:5001"
-  #   - "kind-registry:5000"
-  # Policy service configuration
-  # policyService:
-  #  topic: "wasmcloud.policy"
-  #  changesTopic: "bar"
-  #  timeoutMs: 10000
-  # Additional options to control how the underlying wasmCloud hosts are scheduled in Kubernetes.
-  # This includes setting resource requirements for the nats and wasmCloud host
-  # containers along with any additional pot template settings.
-  # schedulingOptions:
-  # Enable the following to run the wasmCloud hosts as a DaemonSet
-  # daemonset: true
-  # Set the resource requirements for the nats and wasmCloud host containers.
-  # resources:
-  #   nats:
-  #     requests:
-  #       cpu: 100m
-  #  wasmCloudHost:
-  #     requests:
-  #       cpu: 100m
-  # Any additional pod template settings to apply to the wasmCloud host pods.
-  # See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#podspec-v1-core for all valid options.
-  # Note that you *cannot* set the `containers` field here as it is managed by the controller.
-  # podTemplateAdditions:
-  #   spec:
-  #     env:
-  #       - name: HOST_IP
-  #         value: spec.hostIP
-  #  nodeSelector:
```

Comment:
The reason this was all here was to illustrate all of the available options. Should we have another version of this that has all of the potential fields?

Reply:
That's a good idea, maybe having something like `examples/complete-config` might be a good place to show case it

Reply:
makes sense. think we bring it back under a different name. maybe `wasmcloudhostconfig-all-options.yaml` or `wasmcloudhost-annotated.yaml` ?

Reply:
`wasmcloudhostconfig-all-features.yaml` maybe? I'm not too concerned about the name, mostly just that it's something we can point at.

Reply:
I think if it were to go under something like `examples/complete-config/wasmcloud-annotated.yaml` seems like a reasonable name for it